server/controllers/UsersController.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable class-methods-use-this */
+import dotenv from 'dotenv';
+import bcrypt from 'bcryptjs';
+import jwt from 'jsonwebtoken';
+import Controller from './Controller';
+import db from '../models/db';
+
+dotenv.config();
+const secret = process.env.JWT_SECRET;
+
+class UsersController extends Controller {
+  /**
+   * @description Function to generate token
+   *
+   * @param {Object} user
+   *
+   * @return {String} Returned token
+   */
+  static Token(user) {
+    const userDetails = {
```

Comment:
is there a reason why you had to create a new object(user details) since user is already an object. Even if you need to derive new variables from the user object, you can actually destructure.

Reply:
I have always picked some properties from the user when generating token, I didn't realize I was duplicating the user object this time around. I have fixed that.